### DISCUSSION TITLE
Welcome to sessions Discussions!

### DISCUSSION THREAD

**Discussion by caichinger** [swkWien/sessions#126] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by caichinger** [comment]
A thread

**Reply by caichinger** [reply]
Look, this is a subthread

**Reply by caichinger** [reply]
Maybe that's useful to organize conversations.

Don't ask me why issues can't do that... (in GitLab they can)

**Reply by caichinger** [reply]
@ReaSu hast du eine Notification bekommen?

**Reply by ReaSu** [reply]
Ja, oder sogar 4 :grin: 
Also Notifications dass die Diskussion eröffnet ist, und für jeden Eintrag.
Danke, dass du das gleich gestartet hast!

**Reply by caichinger** [reply]
Gerne. Dachte vielleicht ist das schon vor dem nächsten GDCR nützlich. :)

---

Alle die ein bisschen genervt sind, weil sie schon wieder ein Haufen Mail bekommen haben:

Einstellungen sind in https://github.com/settings/notifications

![image](https://github.com/swkWien/sessions/assets/16005912/ad31832f-55a1-466b-a493-36fbeb3c7663)
